framework/Source/PictureInput.swift

```diff
@@ -54,41 +58,78 @@ public class PictureInput: ImageSource {
             let textureLoader = MTKTextureLoader(device: sharedMetalRenderingDevice.device)
             if synchronously {
                 do {
-                    let imageTexture = try textureLoader.newTexture(cgImage:internalImage!, options: [MTKTextureLoader.Option.SRGB : false])
+                    let mtlTexture = try textureLoader.newTexture(cgImage:internalImage!, options: newTextureOptions)
                     internalImage = nil
-                    self.internalTexture = Texture(
-                        orientation: .portrait,
-                        texture: imageTexture,
-                        timingStyle: isTransient ? .transientImage : .stillImage
-                    )
-                    self.updateTargetsWithTexture(self.internalTexture!)
+                    let texture = makeTexture(with: mtlTexture)
+                    self.internalTexture = texture
+                    self.updateTargetsWithTexture(texture)
                     self.hasProcessedImage = true
                 } catch {
                     fatalError("Failed loading image texture")
                 }
             } else {
-                textureLoader.newTexture(cgImage: internalImage!, options: [MTKTextureLoader.Option.SRGB : false], completionHandler: { [weak self] (possibleTexture, error) in
+                textureLoader.newTexture(cgImage: internalImage!, options: newTextureOptions, completionHandler: { [weak self] (possibleTexture, error) in
                     guard let self else { return }
                     guard (error == nil) else { fatalError("Error in loading texture: \(error!)") }
-                    guard let texture = possibleTexture else { fatalError("Nil texture received") }
+                    guard let mtlTexture = possibleTexture else { fatalError("Nil texture received") }
                     self.internalImage = nil
-                    self.internalTexture = Texture(
-                        orientation: .portrait,
-                        texture: texture,
-                        timingStyle: self.isTransient ? .transientImage : .stillImage
-                    )
+
+                    let texture = makeTexture(with: mtlTexture)
+                    self.internalTexture = texture
                     DispatchQueue.global().async{
-                        self.updateTargetsWithTexture(self.internalTexture!)
+                        self.updateTargetsWithTexture(texture)
                         self.hasProcessedImage = true
                     }
                 })
             }
         }
     }
-    
+
+    @available(iOS 13, *)
```

Comment:
We could make the target 13+

Reply:
Hi @yyjim, I've updated the target to `.v13` and removed this `@available` in the latest PR.
Also, I'm going to merge the PR and add the tag `1.4.0` to this repo; thanks!